CHANGELOG.md

```diff
@@ -412,7 +419,8 @@ the [changelog](https://github.com/cosmos/cosmos-sdk/blob/v0.38.4/CHANGELOG.md).
 [#750]: https://github.com/Kava-Labs/kava/pull/750
 [#751]: https://github.com/Kava-Labs/kava/pull/751
 [#780]: https://github.com/Kava-Labs/kava/pull/780
-[unreleased]: https://github.com/Kava-Labs/kava/compare/v0.26.1...release/v0.26.x
+[unreleased]: https://github.com/Kava-Labs/kava/compare/v0.26.2...release/v0.26.x
+[v0.26.2]: https://github.com/Kava-Labs/kava/compare/v0.26.2...v0.26.1
```

Comment:
I think it should be the other way around, older tag first then newer in the url. Some previous versions also link it the opposite way and those don't work like `v0.26.1`

```suggestion
[v0.26.2]: https://github.com/Kava-Labs/kava/compare/v0.26.1...v0.26.2
```

Reply:
Good catch

Reply:
thanks! fixed this and all the other incorrect links